config/config.go

```diff
@@ -395,6 +395,10 @@ type SlaveOptionsConfig struct {
 
 	// SynchroniserEnabled enable this config if MDCB has enabled the synchoniser. If disabled then it will ignore signals to synchonise recources
 	SynchroniserEnabled bool `json:"synchroniser_enabled"`
+
+	// OriginalAPIKeyPath is the original path to the API key in the configuration file. Only used to modify the external KV store in case of API Key Reset.
+	// This is set automatically in afterConfSetup()
+	OriginalAPIKeyPath string `json:"-"`
```

Comment:
why do we want to have `OriginalAPIKeyPath`? this is an historic value and should keep using the new one no matter how many times we rotate the key.
Also, wouldnt this be problematic when generting the docs for config options?